html/Task_3/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 40%;
+  min-width: 450px;
+  margin: 0 auto;
+  padding: 20px;
+  background: lightgray;
+  font-family: "Lato", sans-serif;
+  font-weight: bold;
+  opacity: 0.7;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+}
+
+.row-1 span {
+  font-size: 8em;
```

Comment:
It's better to use `rem`and set base root size to have opportunity to scale all text by change base font